changelog/unreleased/enhancement-app-templates

```diff
@@ -0,0 +1,7 @@
+Enhancement: Custom app templates
+
+The existing apps have been refactored and their common functionality has been extracted into a viewer and an editor app template, which can be easily reused to create custom apps.
+
```

Comment:
Not really accurate technically... we should also probably focus more on user value here: more streamlined functionality and behavior

Auto saving for all editors, unified shortcut handling etc

Reply:
We (as in "you") also managed to boil it into one template instead of a viewer/editor one each ;) changelog updated